### PR TITLE
change

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: docs/whats-new/staging-release-notes.md [hunk 1]
```diff
@@ -2,6 +2,47 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 10 May 2023
+
+Frontend version: .0.0-20230509.1043 | Backend version: 1.61.9-230508.0923
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md]: FE version typo, should be 1.0.0-20230509.1043

Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: fixed

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,47 @@
...
+
+> **Note**: This change only affects Public Officers. Vendors currently do not trigger onboarding through the TechPass portal and do not have a **pending** device.
+
+</details>
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> TechBiz groups are excluded from <i>ListGroups</i> implementation by default.</summary>
+
+TechBiz groups will be hidden by default on TechPass portal. This change is also consistent with TechPass portal's **Groups** page, as we are not displaying the TechBiz groups there. However, if you want to include TechBiz groups via the Automation API, you will need to pass the query parameter <i>includeOnTechBiz=true</i>.
+
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md:25]: TechBiz group is already hidden by default on TP portal currently, so this is not a new change in this release. we can just say:

Groups managed by TechBiz are now excluded from <i>ListGroups</i> Automation API by default. To include them, you will need to pass the query parameter <i>includeOnTechBiz=true</i>

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -2,6 +2,47 @@
...
+**Fixes**
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> Enhancements to FormSG integration with Jira and URL change.
+</summary>
+
+We have made some enhancements to our FormSG to better integrate with Jira. As part of the SGTS ITSM Jira Tool, TechPass will now be using the new FormSG. 
+
+In addition, we have also changed the URL for accessing our technical support for SEED from https://go.gov.sg/techpass-sr to https://go.gov.sg/seed-techpass-support.
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md]: the form is not just for SEED, it's for both TechPass & SEED.

File: docs/whats-new/staging-release-notes.md [hunk 4]
```diff
@@ -2,6 +2,47 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 10 May 2023
+
+Frontend version: 1.0.0-20230509.1043 | Backend version: 1.61.9-230508.0923
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: i see the dev backend version is 1.61.9-230508.0922

Comment (yqyeoh) [reply docs/whats-new/staging-release-notes.md]: @kpslp can trouble u to update per jerome's value

Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: changed

File: docs/whats-new/staging-release-notes.md [hunk 5]
```diff
@@ -2,6 +2,41 @@
...
+
+</details>
+
+**Fixes**
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> Enhancements to FormSG integration with Jira and URL change.
+</summary>
+
+We have made some enhancements to our FormSG to better integrate with Jira. As part of the SGTS ITSM Jira Tool, TechPass will now be using the new FormSG.
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: We don't need to inform the users of our backend changes. Let's include only the URL changes.

Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: updated

**Review by JLeeChan — APPROVED** [review]

**Pull request merged**